server/controllers/artistsController.js

```diff
@@ -1,34 +1,50 @@
 import asyncHandler from "../middleware/asyncHandler.js";
 import Artist from "../models/Artist.js";
+import logger from "../logger.js";
 import ErrorResponse from "../utils/ErrorResponse.js";
 import createObjectFromQuery from "../utils/createObjectFromQuery.js";
 import { createDummyArtist } from "../utils/createDummyData.js";
 
 const findArtist = async (req) => {
+  logger.info(`findArtist with ${req} params`);
```

Comment:
req is a huge object.
if you are using only the body from req, just log the body